### DISCUSSION TITLE
Is it possible to build desktop widget like calendar widget ?

### DISCUSSION THREAD

**Discussion by tashi146** [tauri-apps/tauri#4860] [General]
Hi, 
I am new to desktop application. Did some researched and Tauri seems to blazingly good.  I am planning to build calendar widget, I have seen many widgets in windows where we don't see any application in task bar at the bottom. 
Now my question is
**Is desktop application and widgets are the same? If not, does tauri support creating widget application like calendar, clock etc?**   
Any resource or link would be appreciated

Thanks in advance

**Comment by nonnorm** [comment]
Not sure if there's any other difference between 'applications' and 'widgets', but Tauri does let you hide the window in the taskbar: https://docs.rs/tauri/latest/tauri/window/struct.WindowBuilder.html#method.skip_taskbar.

**Comment by arelove** [comment]
> Hi, 
> I am new to desktop application. Did some researched and Tauri seems to blazingly good.  I am planning to build calendar widget, I have seen many widgets in windows where we don't see any application in task bar at the bottom. 
> Now my question is
> **Is desktop application and widgets are the same? If not, does tauri support creating widget application like calendar, clock etc?**   
> Any resource or link would be appreciated
> 
> Thanks in advance
> 

Of course, you can do it, made a different form of window, and turn on transparency on tauri.conf and let's do that!
